lib/polyn/naming.rb

```diff
@@ -10,6 +10,14 @@ def self.dot_to_colon(str)
       str.gsub(".", ":")
     end
 
+    def self.dot_to_underscore(name)
```

Comment:
All these methods are just straight passthroughs to gsub. Can we just inline them instead? Why do we need separate methods and separate tests for built in library functionality?